src/globus_sdk/utils.py

```diff
@@ -32,7 +36,8 @@ def slash_join(a, b):
     return a + "/" + b
 
 
-def safe_stringify(value):
+# REVIEW: bytes has a str method but that isn't how this converts bytes here.
+def safe_stringify(value: ToStr) -> str:
```

Comment:
I think this comment touches on my point of concern with `ToStr` as a type.

`value: Any` is accurate for what this function does and the broadness of its usage. Plus, it's an internal method in a `utils` module, so it's not like we need an annotation which explains it to users.